itm_adm_mock.py

```diff
@@ -16,12 +12,18 @@ def main():
 
     args = parser.parse_args()
 
-    scene_type = "_random_" if args.r else ""
-    use_db = "_db_" if args.db else ""
-    session_type = args.session[0] if args.session else ""
+    if args.session:
+        if args.session[0] not in ['soartech', 'adept', 'eval']:
```

Comment:
It will pertain to the next one (ITM-150: In training mode, let TA1 session span multiple scenarios).  The reason this works, as currently written, is that we know the training session will be either soartech or adept, not both (like eval sessions, which don't support training mode).  If there's a need for a training session to include a mix of soartech and adept scenarios, then we probably need a new session type, e.g., `training`, and will need some way to configure what scenarios should be run (or maybe it runs all of them).

Related, we may need an eval and training sub-folder for scenarios.  And I believe that training sessions don't need set an alignment target (like eval sessions do), because TA2 will call `get_session_alignment()` multiple times with different alignment targets, as I understand it.